dist/backdraft.js

```diff
@@ -1303,6 +1303,19 @@ _.extend(Plugin.factory, {
       }
     },
 
+    // takes a hash of { columnTitle: columnState, ... }
+    setColumnVisibilities: function(columns) {
+      _.each(columns, function(state, title) {
```

Comment:
so this is `state, title` but the method above (and lower level `columnManager.visibility()`) takes the arguments as `title, state`

It's too bad they get yielded as value, key here.. not sure there is much to be done here short of a manual loop construct instead of `_.each`